darts/tests/explainability/test_shap_explainer.py

```diff
@@ -153,6 +153,10 @@ class ShapExplainerTestCase(DartsBaseTestClass):
         )
     )
 
+    models.append(
```

Comment:
I would avoid appending all the models to a list here, as if you change the list later on, it might be messing up which models are called in the tests.
You could just create the model inside the test.

Reply:
Very good point, I corrected it.